crypto/md5/build.info

```diff
@@ -31,7 +30,6 @@ ENDIF
 # Implementations are now spread across several libraries, so the defines
 # need to be applied to all affected libraries and modules.
 DEFINE[../../libcrypto]=$MD5DEF
-DEFINE[../../providers/libimplementations.a]=$MD5DEF
 DEFINE[../../providers/liblegacy.a]=$MD5DEF
 
 GENERATE[md5-586.s]=asm/md5-586.pl
```

Comment:
Might this and the following lines be the reason for the asm getting included?

Reply:
No.  There's a more generic inclusion of perlasm files in `Configuration/unix-Makefile.tmpl`, for the very simple reason that we're not trying to do an exhaustive reconfiguration for every darn platform.  We *could* have a hard coded list instead, but that comes with a maintenance cost.